yarn.lock

```diff
@@ -16276,6 +16281,8 @@ strip-ansi@^4.0.0:
   version "4.0.0"
   resolved "https://registry.yarnpkg.com/strip-ansi/-/strip-ansi-4.0.0.tgz#a8479022eb1ac368a871389b635262c505ee368f"
   integrity sha1-qEeQIusaw2iocTibY1JixQXuNo8=
+  dependencies:
```

Comment:
It keeps haunting me 😭

Reply:
I've been duckduckgoing and couldn't find anything about this